### ISSUE TITLE
Memory leak caused by static variables

### ISSUE THREAD

**Issue by CROSP** [RiccardoMoro/RMSwitch#11]
Hi, thanks for the library. However it has a potential problem causing memory leaks. 
Here is a Leak Canary dump showing how the memory leak occured. 

Could you fix it please ?
<img width="491" alt="screen shot 2017-10-22 at 3 09 21 pm" src="https://user-images.githubusercontent.com/5072318/31861759-490b2bcc-b73b-11e7-8223-7b2d86eb9fb7.png">

**Comment by YuanTiger** [comment]
Have you solved it? I also encountered the same problem.

**Comment by YuanTiger** [comment]
I will refer to the project source code to my project and add code to the RMSwitch:
```
   public void onDestroy() {
        removeSwitchObservers();
        if (mContainerLayout != null) {
            mContainerLayout.removeAllViews();
            mContainerLayout = null;
        }
        if (sLayoutTransition != null) {
            sLayoutTransition = null;

        }
    }

```

**Comment by RiccardoMoro** [comment]
This has been fixed as part of 1.2.3 release now on jcenter. Sorry for the (really) long wait and the kinda stupid mistake.
